<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainwindowClass</class>
 <widget class="QMainWindow" name="MainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>866</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="table">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>561</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gameStart">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>390</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QLabel" name="numUp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>581</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="numDown">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="test">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>331</width>
      <height>721</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="imgSelection">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>480</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QSlider" name="difficulty">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>600</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="timeRecorder">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="record">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> 个人纪录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>计时器</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>610</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>简单</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>普通</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>微软雅黑</family>
        </font>
       </property>
       <property name="text">
        <string>困难</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
